elasticsearch_async/connection.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, host='localhost', port=9200, http_auth=None,
             connector=aiohttp.TCPConnector(
                 loop=self.loop,
                 verify_ssl=verify_certs,
-                conn_timeout=self.timeout,
+                keepalive_timeout=self.timeout,
```

Comment:
It has moved to `ClientSession` - http://aiohttp.readthedocs.io/en/stable/client_reference.html#aiohttp.ClientSession

`keepalive_timeout` is entirely different functionality and cannot be used for this use case.